【bug】Sysbench writes data and removes the slave node POD mysql container，the master/slave relationship is broken

**Describe the problem**
Sysbench writes data and removes the slave node POD mysql container，the master/slave relationship is broken 

the error as follows:
`[ERROR] FOLLOWER[ID:sample-mysql-0.sample-mysql.default:8801, V:35, E:0].mysql.StartSlave.error[Error 1200: The server is not configured as slave; fix in config file or with CHANGE MASTER TO]`

sysbench command as follows:
`sysbench --mysql-host=sample-leader.default --mysql-port=3306 --mysql-user=radondb_usr --mysql-password=RadonDB@123 --mysql-db=radondb --threads=64 --time=300 --report-interval=10 --tables=64 --table-size=50000 oltp_write_only.lua run`


the yaml as follows:
```
spec:
  backupScheduleJobsHistoryLimit: 6
  metricsOpts:
    enabled: false
    image: 'prom/mysqld-exporter:v0.12.1'
    resources:
      limits:
        cpu: 100m
        memory: 128Mi
      requests:
        cpu: 10m
        memory: 32Mi
  minAvailable: 50%
  mysqlOpts:
    database: radondb
    initTokuDB: false
    password: RadonDB@123
    resources:
      limits:
        cpu: 4
        memory: 8Gi
      requests:
        cpu: 2
        memory: 4Gi
    rootHost: localhost
    rootPassword: ''
    user: radondb_usr
  mysqlVersion: '8.0'
  persistence:
    accessModes:
      - ReadWriteOnce
    enabled: true
    size: 20Gi
  podPolicy:
    affinity: {}
    auditLogTail: false
    busyboxImage: 'busybox:1.32'
    extraResources:
      requests:
        cpu: 10m
        memory: 32Mi
    imagePullPolicy: IfNotPresent
    sidecarImage: 'radondb/mysql80-sidecar:v2.2.1-alpha.3'
    slowLogTail: false
  replicas: 3
  xenonOpts:
    admitDefeatHearbeatCount: 5
    electionTimeout: 10000
    enableAutoRebuild: true
    image: 'radondb/xenon:v2.2.1-alpha.3'
    resources:
      limits:
        cpu: 100m
        memory: 256Mi
      requests:
        cpu: 50m
        memory: 128Mi
```

<!-- Please describe the issue you observed, and any steps we can take to reproduce it: -->

<!-- If applicable, add screenshots to help explain your problem. -->

**To Reproduce**

<!-- What did you do? Describe in your own words. -->

<!-- If possible, provide steps to reproduce the behavior: -->

1.Sysbench writes data
2.removes the slave node POD mysql container

**Expected behavior**

<!-- A clear and concise description of what you expected to happen. -->

**Environment:**

<!--
e.g.
- Kubernetes version: v1.19.5
- Helm version: v3.4.2
- RadonDB MySQL version: https://github.com/radondb/radondb-mysql-kubernetes/commit/56896cd119d9a6017e7f15f1dc0b37b83a720278
-->

- RadonDB MySQL version:
-

--- a/mysqlcluster/container/container.go
+++ b/mysqlcluster/container/container.go
@@ -39,6 +39,29 @@ type container interface {
 	getVolumeMounts() []corev1.VolumeMount
 }
 
+func getStartupProbe(name string) *corev1.Probe {
+	if name == utils.ContainerMysqlName {
+		return &corev1.Probe{
+			Handler: corev1.Handler{
+				Exec: &corev1.ExecAction{
+					Command: []string{
+						"sh",
+						"-c",
+						`if test $(mysql -uroot -h127.0.0.1 -NB -e "SELECT 1") -eq 1; then sed -i "/^RESET SLAVE ALL;/d" /etc/mysql/conf.d/init.sql; fi`,
+					},
+				},
+			},
+			InitialDelaySeconds: 10,
+			TimeoutSeconds:      5,
+			PeriodSeconds:       10,
+			SuccessThreshold:    1,
+			FailureThreshold:    5,
+		}
+	} else {
+		return nil
+	}
+}
+
 // EnsureContainer ensure a container by the giving name.
 func EnsureContainer(name string, c *mysqlcluster.MysqlCluster) corev1.Container {
 	var ctr container
@@ -72,6 +95,7 @@ func EnsureContainer(name string, c *mysqlcluster.MysqlCluster) corev1.Container
 		Ports:           ctr.getPorts(),
 		LivenessProbe:   ctr.getLivenessProbe(),
 		ReadinessProbe:  ctr.getReadinessProbe(),
+		StartupProbe:    getStartupProbe(name),
 		VolumeMounts:    ctr.getVolumeMounts(),
 	}
 }

